src/sql/workbench/parts/dataExplorer/browser/connectionViewletPanel.ts

```diff
@@ -52,7 +50,6 @@ export class ConnectionViewletPanel extends ViewletPanel {
 		@IObjectExplorerService private objectExplorerService: IObjectExplorerService
 	) {
 		super({ ...(options as IViewletPanelOptions), ariaHeaderLabel: options.title }, keybindingService, contextMenuService, configurationService);
-		//this._clearSearchAction = this.instantiationService.createInstance(ClearSearchAction, ClearSearchAction.ID, ClearSearchAction.LABEL, this);
 		this._addServerAction = this.instantiationService.createInstance(AddServerAction,
```

Comment:
This looks to be the only place the ClearSearchAction is actually used so you can just get rid of the class completely